File: backend/Loan/Loan.BLL/DataTransferObjects/LoanDto.cs
using Common.Enum;

namespace Loan.BLL.DataTransferObjects;

public class LoanDto {
    public Guid Id { get; set; }
    
    public Guid UserId { get; set; }
    
    public Guid AccountId { get; set; }
    
    public TariffDto Tariff { get; set; }
    
    public DateTime? LastChargeDate { get; set; }
    
    public CurrencyType CurrencyType { get; set; }
    
    public int Debt { get; set; }
}

File: frontend/web-app-employee/src/services/loan/models/LoanDto.ts
import {CurrencyType} from "../../common/CurrencyType.ts";
import {TariffDto} from "../../tariff/models/TariffDto.ts";

export type LoanDto = {
    id: string
    userId: string
    accountId: string
    tariff: TariffDto
    lastChargeDate?: string
    currencyType: CurrencyType
    sum: number
    debt: number
}
